### PR TITLE
Prob44

### PR DESCRIPTION
**Pull request by logic-legolas**
finished
@rickyc
@griselleong

### CODE REVIEW

File: prob44/prob44.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script type="text/javascript">
+      const calculate=(array)=> {
+        var stats=[];
```
Comment (rickyc) [inline prob44/prob44.html:10]: Let's use ES6 `const` and `lets`

File: prob44/prob44.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        var sum = 0;
+        for (var x = 0; x < array.length; x++) {
+          if (smallest>array[x]) {
+            smallest = array[x];
+          }
+          if (largest<array[x]) {
+            largest = array[x];
+          }
+          sum+=array[x];
+          var average = sum/array.length
```
Comment (rickyc) [inline prob44/prob44.html]: Take a look at this variable and consider the scoping of it.

**Review by rickyc — CHANGES_REQUESTED** [review]

File: prob44/prob44.html (hunk 1 above)
Comment (logic-legolas) [reply prob44/prob44.html:10]: for whatever reason i cant use ES6 on the laptop

File: prob44/prob44.html (hunk 2 above)
Comment (logic-legolas) [reply prob44/prob44.html]: ok

File: prob44/prob44.html (hunk 1 above)
Comment (rickyc) [reply prob44/prob44.html:10]: 😱 That's a problem!

**Review by rickyc — APPROVED** [review]
Alright, you can merge it in for now. You'll need to figure out that computer situation 😅

File: prob44/prob44.html (hunk 1 above)
Comment (logic-legolas) [reply prob44/prob44.html:10]: i know

Comment (logic-legolas) [reply prob44/prob44.html:10]: whenever I do it gives me a "Uncaught SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode" error

**Pull request merged**